open-source-services.md

```diff
@@ -28,19 +28,47 @@ Thus we must address common questions about the requirements:
 
 ### Q: What defines a service?
 
-A service is software operated for the user. Software the user is operating for themself is not considered a service. A service is defined by the context in which the software is operated and all of the assets required to operate the software for the user in that context.
+A service is software operated for the user.
+Software the user is operating for themself is not considered a service.
+A service is defined by the context in which the software is operated and all of the assets required to operate the software for the user in that context.
```

Comment:
This seems like simply a formatting changed mixed in with substantive changes. Did I understand that right?

Reply:
It is only a formatting change, I don't think there is a change in the content itself.

In fact, the reason for the formatting change is to make it easier to read substantive changes.